runtime/pangolin/src/pallets/ecdsa_authority.rs

```diff
@@ -29,7 +29,7 @@ static_assertions::const_assert!(MAX_PENDING_PERIOD > SYNC_INTERVAL);
 
 impl darwinia_ecdsa_authority::Config for Runtime {
 	type ChainId = <Self as pallet_evm::Config>::ChainId;
-	type MaxAuthorities = ConstU32<3>;
+	type MaxAuthorities = ConstU32<7>;
```

Comment:
@echo Please help check this value.

Reply:
This max, can be 5?

Reply:
Yes, but it's just an upper limit. Every authority needs to be added/reviewed by the governance.